tests/e2e/ppo_trainer/run_model_reward.sh

```diff
@@ -20,7 +20,7 @@ TEST_FREQ=${TEST_FREQ:--1}
 # Save & Resume
 RESUME_MODE=${RESUME_MODE:-disable}
 SAVE_FREQ=${SAVE_FREQ:--1}
-TOT_TRAIN_STEPS=${TOT_TRAIN_STEPS:-1}
+TOT_TRAIN_STEPS=${TOT_TRAIN_STEPS:-2}
```

Comment:
Better not to change this default value. Use env to change behavior in specific test.